README.md

```diff
@@ -1,34 +1,145 @@
+This plugin supports a two-way integration between Mattermost and Jira. For a stable production release, please download the latest version [in the Releases tab](https://github.com/mattermost/mattermost-plugin-jira/releases) and follow [these instructions](#2-configuration) for install and configuration.
 
-We are looking for help from our community to complete the development of v2.0 of the Mattermost Jira plugin. If you're interested, review [open Help Wanted issues](https://github.com/mattermost/mattermost-plugin-jira/issues?q=is%3Aissue+is%3Aopen+label%3A%22help+wanted%22) to get started.
+This plugin supports Jira Core and Jira Software products, for Server, Data Center and Cloud platforms.
+
+Support for multiple Jira instances is considered, but not yet supported.
+
+## Table of Contents
+
+ - [1. Features](#1-features)
+ - [2. Configuration](#2-configuration)
+ - [3. Jira v2 Roadmap](#3-jira-v2-roadmap)
+ - [4. Development](#4-development)
+ - [5. Frequently Asked Questions (FAQ)](#5-frequently-asked-questions-faq)
 
 ## 1. Features
 
-Below is a list of features currently supported. Each is considered Beta and may be removed in subsequent updates. We are also aware of a few known issues and are tracking development [in Jira](https://mattermost.atlassian.net/issues/?jql=status%20in%20(%22At%20Risk%22%2C%20Backlog%2C%20%22Future%20Consideration%22%2C%20%22In%20Progress%22%2C%20Open%2C%20Pending%2C%20%22Planned%3A%20Backlog%22%2C%20%22Planned%3A%20Scheduled%22%2C%20Reopened%2C%20Resolved%2C%20Reverted%2C%20%22Selected%20for%20Development%22%2C%20Submitted%2C%20%22To%20Do%22)%20AND%20%22Epic%20Link%22%20%3D%20MM-12474%20ORDER%20BY%20cf%5B10011%5D%20ASC%2C%20created%20DESC).
+### 1.1 Send notifications from Jira to Mattermost
 
-- Send notifications for issue events from Jira to Mattermost with full JQL support, using webhooks. Restricted to Jira System Admins only.
-   - This includes notifications for the following events: issue created; issue transitioned to “Reopened”, “In Progress”, "Submitted" or “Resolved”; issue deleted or closed; comments created, updated or deleted; assignee updated
-- Create Jira issues via Mattermost UI (Desktop App and browser only)
-- Slash commands for
-  - `/jira connect` - Connect to a Jira project and subscribe to events.
-  - `/jira disconnect` - Disconnect from a Jira project and subscribe from events.
-  - `/jira instance` - Manage connected Jira instances. Must have System Admin role in Mattermost. This is for development purposes only, and to be removed prior to 2.0 release.
-  - `/jira transition <issue-key> <state>` - Transition a Jira issue specified by `issue-key`. `state` must be a valid Jira state such as "Done".
+Notify your team of latest updated by sending notifications from your Jira projects to Mattermost channels.
```

Comment:
Notify your team of the latest updates by sending notifications from your Jira projects to Mattermost channels.

Reply:
Updated